BoxBrowseSDK/BoxBrowseSDK/Categories/UIImage+BOXBrowseSDKAdditions.m

```diff
@@ -100,6 +100,36 @@ + (UIImage *)box_genericFolderIcon
     return [UIImage box_iconWithName:@"personal_folder"];
 }
 
++ (UIImage *)box_sharedFolderIcon
```

Comment:
I didn't change the `box_iconForItem:item` and `box_smallIconForItem:item` to use these API because I think it would be better to refactor those 2 meaty methods to avoid all their duplication. That is a far worse duplication than having the same string (e.g. @"shared_folder") referenced in 2 places.